python/foglamp/services/core/server.py

```diff
@@ -70,9 +75,22 @@ class Server:
     core_management_port = 0
     """ Microservice management port of core """
 
-    # TODO: FOGL-655 Get Admin API port from configuration option
-    rest_service_port = 8081
-    """ Admin service port of core """
+    rest_server_port = 0
+
+    is_rest_server_tls_enabled = False
+
+    _REST_API_DEFAULT_CONFIG = {
+        'port': {
+            'description': 'REST API service port',
+            'type': 'integer',
+            'default': '8081'
+        },
+        'isTlsEnabled': {
```

Comment:
The spec [The FogLAMP REST API](https://docs.google.com/document/d/1JJDP7g25SWerNVCxgff02qp9msHbqA9nt3RAFx8-Qng/edit#heading=h.v60x3i70chpa) has a different set of configuration options.

Reply:
@MarkRiddoch  fixed, please check now.